test/StressTests/Corax/Bugs/PostingListTestsExtended.cs

```diff
@@ -18,7 +18,7 @@ public class PostingListTestsExtended(ITestOutputHelper output) : NoDisposalNoOu
     public void CanDeleteAndInsertInRandomOrder(int seed, int size) => CanDeleteAndInsertInRandomOrderBase(seed, size);
 
     [RavenMultiplatformTheory(RavenTestCategory.Corax | RavenTestCategory.Voron, RavenArchitecture.X64)]
-    [InlineData(1477187726, true, 1828658)]
+    [InlineData(1477187726, true, 1828658, Skip = "Too big")] // NativeList can contain only up to int.MaxValue / sizeof(long), which is smaller than maxSize of this test 
     [InlineDataWithRandomSeed(20000000, false)]
```

Comment:
Instead of this, ensure that the test is going to handle that case specifically and write a test itself that would trigger this for the NativeList instead.